cinema/serializers.py

```diff
@@ -125,10 +128,10 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
-            ValidationError
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
+            ValidationError,
```

Comment:
Ensure that the `validate_ticket` method in the `Ticket` model is correctly handling the validation logic. This method is crucial for maintaining data integrity when creating or updating tickets.